### ISSUE TITLE
Update peer dependencies for Angular 8

### ISSUE THREAD

**Issue by MicMicMon** [dbfannin/ngx-logger#122]
`ng update --all` prevents some updates because of `ngx-logger` dependencies:
```
Package "ngx-logger" has an incompatible peer dependency to "@angular/common" (requires "^6.0.0 || ^7.0.0", would install "8.0.0")
```
Would it be possible to update the specs to allow version `8`?
Thanks for the awesome package!

**Comment by bmtheo** [comment]
Yes that would be great, just for info I still managed to update to Angular8 without touching ngx-logger
The logger seems to work fine for now

PS : My version of ngx-logger is 3.3.11 (not the latest which is 3.3.13 as of now)

**Comment by dbfannin** [comment]
fixed in ngx-logger@3.4.1

**Comment by bmtheo** [comment]
Thanks for the quick fix, it works fine !
cheers
